### DISCUSSION TITLE
E5 subscription을 다시 받을 수 있는지 확인 부탁드립니다.

### DISCUSSION THREAD

**Discussion by kim-yeon-hong** [fusiondevkr/pl900#81] [1 Q&A]
1년 전 온라인 강의를 듣기 시작했습니다. 

강의 중에 E5 subscription을 신청하라고 하던데 지금 들어가보니 안되는 것 같습니다.

E5 subscription을 다시 받을 수 있는지 확인 부탁드립니다. 

감사합니다. 

yeonhong.kim@dkbmc.com 김연홍

**Comment by justinyoo** [comment]
안녕하세요, 기존의 E5 구독이 만료가 됐다면 수동으로 삭제하고 새로 구독을 신청해 보세요. 만약 새로 구독이 만들어지지 않는다면 악의적인 용도로 구독을 만드는 계정들이 많아서 요즘 그거 필터링한다고 한번에 안되는 경우가 있다고 하니, 에러 메시지를 보시고 며칠 후에 다시 시도해 보시면 될 겁니다.

**Reply by kim-yeon-hong** [reply]
감사합니다. ^^;
